### ISSUE TITLE
The Tabs bar cannot load 

### ISSUE THREAD

**Issue by weiliank** [GeekyAnts/NativeBase#548]
## react-native, react and native-base version
"native-base": "^2.0.8",
"react": "15.4.2",
"react-native": "0.41.2"

i am getting this error : "Invariant Violation" with this description below even after i using the latest master github repo. Any idea? I tried googling around but i can't find any possible solution.

"Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object. Check the render method of 'ResumeHome'"

My sample code:
```
render() {
        return (
            <Container>
                    <Header>
                        <Left>
                            <Button transparent onPress={Actions.pop}>
                               <Icon ios='ios-arrow-back' android='md-arrow-back'/>
                            </Button>
                        </Left>
                        <Body>
                            <Title>Resume</Title>
                        </Body>
                        <Right/>
                    </Header>
                    
                    <Content>
                        <StyleProvider style={getTheme(customTheme)}>
                             <Tabs>
                                <Tab heading="View">
                                    <TabOne />
                                </Tab>
                                <Tab heading="Configure">
                                    <TabTwo />
                                </Tab>
                            </Tabs>
                        </StyleProvider>
                    </Content>
                </Container>
        )
    }
```

**Comment by weiliank** [comment]
I tried removing the `<Tabs>` and it works but it links back to issue #541 (I'm using the latest github commit)

Below is the warning message shown when I tried to navigate to the Tabs page.
![warning](https://cloud.githubusercontent.com/assets/7866665/23172534/051e8ecc-f891-11e6-9bf7-ce286321cfeb.PNG)

**Comment by sankhadeeproy007** [comment]
Ideally the tab should be outside Content

**Comment by weiliank** [comment]
Nope it's nothing do with `<Content>` 

Any specific adjustment on `<Tabs>`?  cause  `<Tabs>` is the one making all this behind.

**Comment by weiliank** [comment]
"native-base": "^2.0.8",
"react": "15.4.2",
"react-native": "0.41.2"

Yes i did upgrade from an older version of native-base and i also eject the variables based on the instructions by running `ejectTheme.js`

Is there anymore links that i can refer on ejecting variables?

**Comment by sankhadeeproy007** [comment]
Some variables have been updated. You might have to eject again for it to work properly.

**Comment by weiliank** [comment]
I followed the docs page instructions exactly. (http://nativebase.io/docs/v2.0.0/components#tabs)

Mind explaining the eject part? :) Thanks!

**Comment by sankhadeeproy007** [comment]
Run `node node_modules/native-base/ejectTheme.js` again. It copies the theme variables and styles to the root folder. Please keep in mind that if you have changed some variables or theme styles, you might have to do it again.

**Comment by sankhadeeproy007** [comment]
Fixed in #541.
